src/main/webapp/cluster-view/jquery.nouislider.ispn.css

```diff
@@ -0,0 +1,166 @@
+
```

Comment:
This is vendor css, correct? In that case we should place it somewhere else! If you create directive using UI-slider you could put this css in same directory with directive.